config/log4j2.properties

```diff
@@ -61,13 +64,29 @@ appender.consoleStderr.filter.acceptGteWarn.level = WARN
 appender.consoleStderr.filter.acceptGteWarn.onMatch = ACCEPT
 appender.consoleStderr.filter.acceptGteWarn.onMismatch = DENY
 
+appender.routing.name = routingAppender
+appender.routing.type = Routing
+appender.routing.purge.type = IdlePurgePolicy
+appender.routing.purge.timeToLive = 60
+appender.routing.route.type = Routes
+appender.routing.route.pattern = $${ctx:ST-JID}
+appender.routing.route.system.type = Route
+appender.routing.route.system.key = $${ctx:ST-JID}
+appender.routing.route.system.ref = fileAppender
+appender.routing.route.job.type = Route
+appender.routing.route.job.appender.type = File
+appender.routing.route.job.appender.name = job-${ctx:ST-JID}
+appender.routing.route.job.appender.fileName = ${file_path}/job-${ctx:ST-JID}.log
+appender.routing.route.job.appender.layout.type = PatternLayout
+appender.routing.route.job.appender.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-30.30c{1.}] [%t] - %m%n
+
 appender.file.name = fileAppender
 appender.file.type = RollingFile
 appender.file.fileName = ${file_path}/${file_name}.log
 appender.file.filePattern = ${file_path}/${file_name}.log.%d{yyyy-MM-dd}-%i
 appender.file.append = true
 appender.file.layout.type = PatternLayout
-appender.file.layout.pattern = [%X{ST-JID}] %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-30.30c{1.}] [%t] - %m%n
+appender.file.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-30.30c{1.}] [%t] - %m%n
```

Comment:
If we do not supported `[%X{ST-JID}]`. We should make sure this PR be merged before 2.3.8 release.

Reply:
Updated docs